mantidproject/mantid
# Sans usability fixes

**Description of work.**
This PR tidies up some minor issues with the ISIS SANS GUI


**To test:**

* Check that the tab ordering in the GUI is sensible. 
* Check that the wavelength slice and event slice tooltips make sense.


Fixes #22776. 

Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
